src/index.html

```diff
@@ -17,16 +17,27 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="input input--large"
+      data-qa="big"
+    >
+      <input
+        class="input__field input__field--large"
```

Comment:
The `input` element is missing the `name` attribute. This attribute is important for accessibility and for form data submission. Even if you're not submitting the form, it's a good practice to always include a `name` attribute.